src/objects/PointCloud.js

```diff
@@ -80,18 +80,15 @@ THREE.PointCloud.prototype.raycast = ( function () {
 			if ( attributes.index !== undefined ) {
 
 				var indices = attributes.index.array;
-				var offsets = geometry.offsets;
 
-				if ( offsets.length === 0 ) {
+				if ( geometry.drawcalls.length === 0 ) {
 
-					offsets.push( {
-						start: 0,
-						count: indices.length,
-						index: 0
-					} );
+					offsets.addDrawCall( 0, indices.length, 0 );
 
```

Comment:
I do not think `offsets` is defined. Also, I do not think we want to modify the geometry here. I thought the intent was to create a temporary `offsets` array if `geometry.drawcalls` was empty.